torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -1048,6 +1048,49 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 output_shape = node.meta["val"].size()
                 return insert_sharded_nhwc_to_nchw(g, output_tensor, output_shape)
 
+            if node.target == torch.ops.aten.slice_scatter.default:
+                tensor, src_tensor, dim, start, end, *step = args
+
+                tensor_shape = tensor.meta["val"].size()
+                src_tensor_shape = src_tensor.meta["val"].size()
+
+                # nothing to slice, result tensors == src_tensor
+                if tensor_shape == src_tensor_shape:
+                    return src_tensor
+
+                # slice_scatter could be concat([pre_slice_tensor, src_tensor, post_slice_tensor])
+                rank = len(tensor_shape)
+                end = min(tensor_shape[dim], end) if end >= 0 else (end + tensor_shape[dim])
+                [step] = step or [1]
+
+                if step != 1 or dim >= rank:
+                    return None
+
+                tensors_list = []
+                # pre_slice_tensor
+                if start > 0:
+                    slice_start = np.zeros(rank, dtype=int)
+                    slice_end = list(tensor_shape)
+                    slice_end[dim] = start
+                    tensors_list.append(g.call_function(ttnn.slice, (tensor, [*slice_start], [*slice_end])))
```

Comment:
If both `slice_start` and `slice_end` are list, we can directly pass them?

```suggestion
                    tensors_list.append(g.call_function(ttnn.slice, (tensor, slice_start, slice_end)))
```

Reply:
I have revised it.